py_trees/behaviours.py

```diff
@@ -736,3 +737,47 @@ def update(self) -> common.Status:
                 "|".join(["T" if result else "F" for result in results])
             )
             return common.Status.FAILURE
+
+
+class ProbabilisticBehaviour(behaviour.Behaviour):
+    """
+    Return a status based on a probability distribution.
+
+    That is, :data:`~py_trees.common.Status.SUCCESS` with probability success_probability,
+    :data:`~py_trees.common.Status.FAILURE` with probability failure_probability and
+    :data:`~py_trees.common.Status.RUNNING` with probability running_probability.
+    
+    If unspecified - a uniform distribution will be used.
+
+    Args:
+        name: name of the behaviour
+        weights: 3 probabilities that correspond to returning :data:`~py_trees.common.Status.SUCCESS`,
+        :data:`~py_trees.common.Status.FAILURE` and :data:`~py_trees.common.Status.RUNNING` respectively.
```

Comment:
You need 3 spaces prior to `:data:...` here to prevent sphinx from complaining.

Reply:
That was a very misleading warning!

Reply:
Thanks for the sharp eye! fixed